epub33/rs/index.html

```diff
@@ -2067,92 +2073,193 @@ <h2>Processing structural semantics</h2>
+			</ul>
+
+			<p>To <dfn>obtain an expanded URL</dfn>, apply the following steps:</p>
+
+			<ol class="algorithm">
+				<li>
+					<p>Let <var>baseURL</var>, <var>expandedURL</var>, <var>propertyPrefix</var>, and
+							<var>propertyName</var> be empty [=strings=].</p>
+					<details class="explanation">
+						<summary>Explanation</summary>
+						<p>In this algorithm:</p>
+						<ul>
+							<li><var>baseURL</var> will hold the base URL associated with the value, whether this URL is
+								assigned by the EPUB creator, corresponds to a default vocabulary, or comes from a
+								reserved prefix.</li>
+							<li><var>expandedURL</var> will hold the final URL that results from the concatenation of
+								the base URL and property name.</li>
+							<li><var>propertyPrefix</var> will hold the prefix for the property (i.e., the value before
+								the colon). Properties from a default vocabulary will not have a prefix, but all others
+								will.</li>
+							<li><var>propertyName</var> will hold the name of the property (i.e., the value after the
+								prefix). All valid properties require a name.</li>
+						</ul>
+					</details>
+				</li>
+				<li>
+					<p>Obtain the values of <var>propertyPrefix</var> and <var>propertyName</var> as follows:</p>
+					<ol>
 						<li>
-							<p id="confreq-rs-vocab-property-reference">If the property consists only of a reference,
-								concatenate the prefix URL associated with the <a href="#sec-default-vocab">default
-									vocabulary</a> to the reference.</p>
+							<p>If <var>value</var> does not contain a colon, set <var>propertyName</var> to
+									<var>value</var> and continue processing at the next step.</p>
+							<details class="explanation">
+								<summary>Explanation</summary>
+								<p>If a <code>property</code> value does not have a colon, it does not have a prefix. In
+									this case, the value is drawn from the default vocabulary.</p>
+							</details>
 						</li>
 						<li>
-							<p>
-								<span id="confreq-rs-vocab-property-prefixed">If the property consists of a prefix and
-									reference, concatenate the URL defined for the prefix to the reference.</span>
-								<span id="confreq-rs-vocab-property-invalid-prefix">If the EPUB creator has not defined
-									a matching prefix, and it is not a <a data-cite="epub-33#sec-reserved-prefixes"
-										>reserved prefix</a> [[epub-33]], the property is invalid and reading systems
-									MUST ignore it.</span>
-							</p>
+							<p>Otherwise, if <var>value</var> contains a single colon:</p>
+							<ol>
+								<li>
+									<p>If <var>value</var> begins or ends with a colon, the value is invalid. Return <a
+											data-cite="infra#nulls">null</a>.</p>
+								</li>
+								<li>
+									<p>Otherwise, let <var>tokens</var> be the result of [=strictly split a
+										string|splitting=] <var>value</var> using a colon (U+003A) as the delimiter. Set
```

Comment:
Here again I do not see the infra reference in my local copy... :-(

Reply:
Must be a glitch either locally for you or that's been resolved now.